userguide/storage.rst

```diff
@@ -1295,6 +1295,23 @@ These permissions control the actions users can perform on the dataset
 contents. ACLs are typically used to manage user interactions with
 :ref:`shared datasets <Sharing>`.
 
+The ACL for a new file or directory is typically determined by the
+parent directory ACL. An exception is when there are no *File* or
+*Directory Inherit* :ref:`flags <ACE Inheritance Flags>` in the parent
+ACL :literal:`owner@`, :literal:`group@`, or :literal:`everyone@`
+entries. In this case the non-inheriting entries for these principals
```

Comment:
In which case?  This is non-specific.  This is talking about the exception, but is not clear about it.